README.md

```diff
@@ -192,6 +192,76 @@ In order to package libraries when connected to an Orchestrator instance, ensure
 
 Do you have any questions regarding the plugin? Ask them [here](https://connect.uipath.com/marketplace/components/jenkins-plugin-for-uipath-public-preview/questions).
 
+## Troubleshooting
+#### [Unauthorized error](#markdown-unauthorized-error)
+#### [Forbidden error](#markdown-forbidden-error)
+#### [Folder/environment not found](#markdown-folder-not-found)
+#### [Package already exists (Conflict)](#markdown-package-already-exists)
+#### [Workspace in Jenkins default installation location](#markdown-workspace-not-accessible)
+#### [Failed to deploy a package which contains non-Latin characters in project name / path to project](#markdown-specify-file-encoding)
+
+### Unauthorized error
+If using basic authentication:
+* ensure the correctness of the username-password combination on the web login
+* if federated authentication is enabled, make sure your write the username in the task as “DOMAIN\user”
+
+If using token authentication:
+* For automating staging.uipath.com or alpha.uipath.com, please reach out to Alvin Stanescu, as some additional configuration might be required
+* Regenerate the token and use new credentials
+
+If authenticating against on an on-premise Orchestrator you might receive this error as a result of the certificate used for the Orchestrator not being valid. This might mean that it has the wrong CN or other validation issues. Please check the certificate if the above mentioned are correct.
```

Comment:
For the last part -> remove the Please ..., instead adding something along the lines of `Ensure that the Orchestrator certificate is valid and that the machine running the job trusts the Orchestrator certificate in case you are using a self-signed certificate.`